openmc/deplete/stepresult.py

```diff
@@ -232,15 +224,15 @@ def get_material(self, mat_id):
                 f'values are {list(self.volume.keys())}'
             ) from e
         for nuc, _ in sorted(self.index_nuc.items(), key=lambda x: x[1]):
-            atoms = self[0, mat_id, nuc]
+            atoms = self[mat_id, nuc]
             if atoms <= 0.0:
                 continue
             atom_per_bcm = atoms / vol * 1e-24
             material.add_nuclide(nuc, atom_per_bcm)
         material.volume = vol
         return material
 
-    def export_to_hdf5(self, filename, step):
+    def export_to_hdf5(self, filename, step, write_rates=True):
```

Comment:
do we want `write_rates=True` here when we've specified it to default to `False` elsewhere?